<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBootloader</class>
 <widget class="QDialog" name="DialogBootloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>更新程序</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>固件路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btSelectFile">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btStartRefreshApp">
          <property name="text">
           <string>更新程序</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="teOutput"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btStartRefreshApp</tabstop>
  <tabstop>btSelectFile</tabstop>
  <tabstop>leFilePath</tabstop>
  <tabstop>teOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
